Crash report "Caused by: java.lang.SecurityException: Permission Denial: reading com.android.providers.media.MediaProvider uri content://media/external/images/media/525327 from pid=1686, uid=10315 requires android.permission.READ_EXTERNAL_STORAGE, or grantUriPermission()"

UUID: 164bdcec7777d8a924fe0e5671d5f0d1
Build flavor: dev
Version name: v4.11.1-13878b2
OS info: Android 7.0, sdk version: 24
Report description: No title
Logs:

```
E/Kuroba-dev | UNCAUGHT: java.lang.RuntimeException: Failure delivering result ResultInfo{who=null, request=2, result=-1, data=Intent { dat=content://media/external/images/media/525327 }} to activity {com.github.adamantcheese.chan.dev/com.github.adamantcheese.chan.StartActivity}: java.lang.SecurityException: Permission Denial: reading com.android.providers.media.MediaProvider uri content://media/external/images/media/525327 from pid=1686, uid=10315 requires android.permission.READ_EXTERNAL_STORAGE, or grantUriPermission()
E/Kuroba-dev | UNCAUGHT: 	at android.app.ActivityThread.deliverResults(ActivityThread.java:4166)
E/Kuroba-dev | UNCAUGHT: 	at android.app.ActivityThread.handleSendResult(ActivityThread.java:4209)
E/Kuroba-dev | UNCAUGHT: 	at android.app.ActivityThread.-wrap20(ActivityThread.java)
E/Kuroba-dev | UNCAUGHT: 	at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1571)
E/Kuroba-dev | UNCAUGHT: 	at android.os.Handler.dispatchMessage(Handler.java:102)
E/Kuroba-dev | UNCAUGHT: 	at android.os.Looper.loop(Looper.java:241)
E/Kuroba-dev | UNCAUGHT: 	at android.app.ActivityThread.main(ActivityThread.java:6217)
E/Kuroba-dev | UNCAUGHT: 	at java.lang.reflect.Method.invoke(Native Method)
E/Kuroba-dev | UNCAUGHT: 	at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:865)
E/Kuroba-dev | UNCAUGHT: 	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:755)
E/Kuroba-dev | UNCAUGHT: Caused by: java.lang.SecurityException: Permission Denial: reading com.android.providers.media.MediaProvider uri content://media/external/images/media/525327 from pid=1686, uid=10315 requires android.permission.READ_EXTERNAL_STORAGE, or grantUriPermission()
E/Kuroba-dev | UNCAUGHT: 	at android.os.Parcel.readException(Parcel.java:1688)
E/Kuroba-dev | UNCAUGHT: 	at android.database.DatabaseUtils.readExceptionFromParcel(DatabaseUtils.java:183)
E/Kuroba-dev | UNCAUGHT: 	at android.database.DatabaseUtils.readExceptionFromParcel(DatabaseUtils.java:135)
E/Kuroba-dev | UNCAUGHT: 	at android.content.ContentProviderProxy.query(ContentProviderNative.java:427)
E/Kuroba-dev | UNCAUGHT: 	at android.content.ContentResolver.query(ContentResolver.java:532)
E/Kuroba-dev | UNCAUGHT: 	at android.content.ContentResolver.query(ContentResolver.java:473)
E/Kuroba-dev | UNCAUGHT: 	at com.github.adamantcheese.chan.ui.helper.ImagePickDelegate.onActivityResult(ImagePickDelegate.java:203)
E/Kuroba-dev | UNCAUGHT: 	at com.github.adamantcheese.chan.StartActivity.onActivityResult(StartActivity.java:545)
E/Kuroba-dev | UNCAUGHT: 	at android.app.Activity.dispatchActivityResult(Activity.java:6958)
E/Kuroba-dev | UNCAUGHT: 	at android.app.ActivityThread.deliverResults(ActivityThread.java:4162)
E/Kuroba-dev | UNCAUGHT: 	... 9 more
E/Kuroba-dev | UNCAUGHT: ------------------------------
E/Kuroba-dev | UNCAUGHT: END OF CURRENT RUNTIME MESSAGES
E/Kuroba-dev | UNCAUGHT: ------------------------------
E/Kuroba-dev | UNCAUGHT: Android API Level: 24
E/Kuroba-dev | UNCAUGHT: App Version: v4.11.1-13878b2
E/Kuroba-dev | UNCAUGHT: Development Build: Yes
E/Kuroba-dev | UNCAUGHT: Phone Model: Sony SO-02H
```

--- a/Kuroba/app/src/main/java/com/github/adamantcheese/chan/ui/helper/ImagePickDelegate.java
+++ b/Kuroba/app/src/main/java/com/github/adamantcheese/chan/ui/helper/ImagePickDelegate.java
@@ -118,6 +118,7 @@ public class ImagePickDelegate {
             Intent newIntent = new Intent(Intent.ACTION_GET_CONTENT);
             newIntent.addCategory(Intent.CATEGORY_OPENABLE);
             newIntent.setPackage(info.activityInfo.packageName);
+            newIntent.addFlags(Intent.FLAG_GRANT_READ_URI_PERMISSION);
             newIntent.setType("*/*");
 
             intents.add(newIntent);

